src/Packetery/Module/Checkout.php

```diff
@@ -1304,13 +1312,25 @@ private function getPostDataIncludingStoredData( string $chosenShippingMethod, i
 
 		if (
 			empty( $checkoutData[ Order\Attribute::ADDRESS_IS_VALIDATED ] ) &&
-			! empty( $savedCarrierData[ Order\Attribute::ADDRESS_IS_VALIDATED ] )
+			! empty(
```

Comment:
To seskupení podmínek do závorky je zbytečné. Špatné použití empty. Ale je to jedno, protože v případě car delivery není ADDRESS_IS_VALIDATED nastaveno, takže podmínka ošetřuje situaci, která nemůže nastat.

Reply:
Odstraneno. Format upraven.